flink-architecture-tests/flink-architecture-tests-base/src/test/java/PredicatesTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.junit.jupiter.api.Test;
+
+import static org.apache.flink.architecture.common.Predicates.getClassSimpleNameFromFqName;
+import static org.assertj.core.api.Assertions.*;
```

Comment:
@afedulov I think this `.*` is causing the compile step to fail with a Checkstyle error

Reply:
https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=53704&view=logs&j=52b61abe-a3cc-5bde-cc35-1bbe89bb7df5&t=54421a62-0c80-5aad-3319-094ff69180bb

Reply:
@bvarghese1 indeed

Reply:
@zentol please merge this hotfix https://github.com/apache/flink/pull/23522/files